starter-code/css/main.css

```diff
@@ -0,0 +1,76 @@
+body {
+  font-weight: 200;
+  color: #333333;
+  padding: 0 25px 0 25px;
+  width: 1000px;
+}
+html {
```

Comment:
It's not best practice to add styles to the html tag since that would get applied to things in the `<head>` which never need CSS. It would be better to apply these styles to the `<body>`.